chichiwang/exercise-js
# Create AnimatedLogo component with spin animation

## Description
Linked to Issue: #81
Create a variant of the `Logo` component that is animated, for use in splash animations/splash screens. `AnimatedLogo` is created, using the same stylesheet module as `Logo`, utilizing a spin [keyframe animation](https://developer.mozilla.org/en-US/docs/Web/CSS/@keyframes).

## Changes
* Updated `pages/components/Logo/index.tsx` to accommodate the style structure required for animating the logo
  * Add a container div to wrap each hex stripe (`.hexMaroonContainer`, `.hexRedContainer`, `.hexOrangeContainer`, `.hexBlueContainer`
* Update test snapshots for every page utilizing the default layout (therefore utilizing the CSS `Logo` in the nav bar)
  * `pages/Counter`
  * `pages/ErrorPage`
  * `pages/Home`
  * `pages/Layout`
  * `pages/NotFound`
* Add a component `pages/components/Logo/AnimatedLogo`, a variant of the Logo with a spin animation applied to it
* Replace the `Logo` component in `pages/Home/index.tsx` with the new `AnimatedLogo` component

## Steps to QA
* Pull down and switch to this branch
* Run the application (`npm run watch`/`npm run dev`/`npm run prod`/`npm start`) and verify no errors either on the server or in the browser
  * Verify that the animation looks sharp
